Server/src/controllers/authenticationController.ts

```diff
@@ -51,6 +51,23 @@ export class AuthenticationController {
         }
     }
 
+    async createPasswordResetRequest(request: Request, response: Response, next: NextFunction): Promise<Response> {
+        this.invalidResponse = this.validatePasswordResetRequest(request);
+        if (this.invalidResponse) {
+            return response.status(400).json("Request is invalid, Email attribute is missing");
+        }
+        else {
+            let requestParams: any = { ...request.body };
+            let passwordResetInfo: IPasswordResetInformation = requestParams;
+            let res: any = await this.callServiceForPasswordReset(passwordResetInfo, response, next);
+            return res;
+        }
+    }
+
+    private validatePasswordResetRequest(request: Request): boolean {
+        return !request.body.hasOwnProperty('email');
```

Comment:
shouldnt this be inverted? if email is present this will invalidate the request?

Reply:
also change this to return request.body.email

Reply:
Na, can't be inverted, the value returned is stored in `invalidResponse`, so should be good